src/pages/Users/helper.tsx

```diff
@@ -0,0 +1,25 @@
+export const FittingDataUserPage = (usersDataResponse) => {
+   const tableData = {
```

Comment:
you lost me on the nested `data`  prop, I don't even care anymore wtf does third `data` mean